tests/unit/test_helpers.py

```diff
@@ -499,6 +500,48 @@ def test_ab_init_active(self):
             ab_init(response)
             response.set_cookie.assert_not_called()
 
+    def test_get_extra_labels(self):
+        """Test extra_labels extraction from form object."""
+        with webcompat.app.test_request_context('/issues/new', method='POST'):
+
+            webcompat.app.preprocess_request()
+
+            self.assertEqual(get_extra_labels(
+                {'extra_labels': '["type-marfeel", "browser-fenix"]'}),
+                ['type-marfeel', 'browser-fenix']
+            )
+            self.assertEqual(get_extra_labels({'extra_labels': '[]'}), [])
+            self.assertEqual(get_extra_labels({}), None)
+            self.assertEqual(get_extra_labels({'extra_labels': ''}), None)
+
+    def test_get_extra_labels_for_experiment(self):
+        """Test extra_labels extraction from form object if
+        experiment is active."""
```

Comment:
nit: one line for the docstring.

Reply:
I had it in one line, but the linter complained that it's too long

Reply:
You can fix that with a magic `  # noqa` comment, but what would you recommend for a better docstring here @karlcow?

Reply:
yes :)

```
"""Test  extra_labels  extraction when A/B experiment is active.

And if you need to add more prose. Jump one line and add more
stuff. :)
"""
```

see https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

(A couple of hours later… doh… forgotten to push comment)